src/main/kotlin/org/jitsi/jibri/webhooks/v1/README.md

```diff
@@ -0,0 +1,46 @@
+# Jibri 'Webhooks'
+
+Jibri supports being configured with a list of base URLs on which it will hit certain endpoints with data.  These are
+not "true" webhooks as the endpoints are hard-coded, instead Jibri defines a "contract" which it expect a subscriber
+to implement at the given base URL.  Information about this contract is below.
+
+### Status updates
+Jibri pushes status updates consisting of its "busy status" (whether it is busy or idle) and its health.
+
+URL: `/v1/status`
```

Comment:
Just this morning I thought that we need to somehow assert that an "real" Jibri is triggering these webhooks. We will probably want to use ASAP JWTs like we've done for other things. Shouldn't block this PR but reading this reminded me of it.